graal-common.json

```diff
@@ -4,5 +4,5 @@
     "overlay": "b84cafb1bb9555b3b13443815280c85df77cfc75"
   },
 
-  "mx_version" : "6.9.1"
+  "mx_version" : "6.9.2"
```

Comment:
That's the workaround: https://github.com/graalvm/mx/commit/a4cac579e721832274f79382e435da8e0ae5465b#diff-9310cdec52557e488a09f6cde609aa14e0c06acf91b3cca6893236ee1b5093ad

Reply:
@lewurm how do you make this workaround work? The root `graal-common.json` in master does not set any `mx_version`. I've tried with the latest mx `6.9.11` and the issue still happens. Only manually patching `suite.py` as per https://gist.github.com/lewurm/03dee75a9b073a7b04692fd6a89b61a1 works.

Reply:
@galderz I reverted the workaround in `mx` 6.9.10, and added a proper fix in `native-image`, see https://github.com/oracle/graal/pull/5226

Sorry for not keeping you in the loop. Does it work for you if you apply the `native-image` patch?